rmgpy/solver/baseTest.py

```diff
@@ -66,15 +84,28 @@ def testSurfaceInitialization(self):
         reactionSystem = self.rmg.reactionSystems[0]
         reactionSystem.attach(self.listener)
         reactionModel = self.rmg.reactionModel
-        
+
         coreSpecies = reactionModel.core.species
         coreReactions = reactionModel.core.reactions
         surfaceSpecies = [coreSpecies[7],coreSpecies[6]] 
         surfaceReactions = [coreReactions[0],coreReactions[2],coreReactions[3]]
-        
-        reactionSystem.initializeModel(coreSpecies,coreReactions,
-                                       reactionModel.edge.species,reactionModel.edge.reactions,surfaceSpecies,surfaceReactions)
-        
+        edgeSpecies = []
+        edgeReactions = []
+        pdepNetworks = []
+        atol = 1e-16
+        rtol = 1e-8
+        sensitivity = False
+        sens_atol = 1e-6
+        sens_rtol = 1e-4
+        filterReactions = False
+        unimolecularFilterFit = self.rmg.reactionSystems[0].unimolecularFilterFit
+        bimolecularFilterFit = self.rmg.reactionSystems[0].bimolecularFilterFit
+        conditions = {}
+
+        reactionSystem.initializeModel(coreSpecies, coreReactions, edgeSpecies, edgeReactions, surfaceSpecies,
```

Comment:
I think you can use keyword arguments here so that you don't have to add the default values for things like `atol` in the previous lines